frontend/src/components/Form/Toast.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { FC } from "react";
+import styled, { css } from "styled-components";
+
+interface iToast {
+  type: string;
+  isActive?: boolean;
+  children?: React.ReactNode;
+  isVisible?: boolean;
+}
+const Toast: FC<iToast> = ({ type, isActive, children }) => {
+  return (
+    <Container type={type} isVisible={isActive}>
+      {children}
+    </Container>
+  );
+};
+
+const Container = styled.section`
```

Comment:
I would say; remove the `isVisible` from the `interface`. This way you have a clear separation between PROPS and STYLED COMPONENTS props.

I would only use the `iToast` interface if the PROPS and styled components PROPS share the same name. 
And if that was the case I would suggest using `Pick` utility to pick only the prop that you're using.

As `isVisible` is only a styled component prop this should be enough:
```javascript
const Container = styled.section<{ isVisible: boolean}>`
```

or 

```javascript
${(p: boolean) =>
```